portality/scripts/prune_marvel.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+import requests
+
+
+def generate_delete_pattern():
+    now = datetime.now()
+    delete_date = now - relativedelta(months=3)
+
+    return ".marvel-{0}*".format(delete_date.strftime('%Y.%m'))
+
+
+def main():
+    delete_pattern = generate_delete_pattern()
+    print "To delete: {0}".format(delete_pattern)
+
+    if len(delete_pattern) < 8:
```

Comment:
Does this mean that the delete pattern is shorter than the string ```.marvel-*```? That's 9 characters. Or have I missed the point of this?

Reply:
That is the idea, yes. I think this kind of check should be done in the tests, now that we have tests to verify the behaviour of the code. In fact we know what the length of the output must be: 16. I've documented that with a new unit test assertion.

Reply:
(and removed this runtime exception)